app/assets/javascripts/govuk_publishing_components/components/accordion.js

```diff
@@ -31,6 +31,14 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
     this.upChevonIconClass = 'gem-c-accordion-nav__chevron'
     this.downChevonIconClass = 'gem-c-accordion-nav__chevron--down'
 
+    // component content pulled from translated data attributes
```

Comment:
Just to be consistent with your other comments:

```suggestion
    // Component content pulled from translated data attributes
```